### ISSUE TITLE
Not allowed because you don't have permission to add files in that folder

### ISSUE THREAD

**Issue by ThePatrickHe** [owncloud/ocis#6614]
Fedora 37 Server, with docker.
Fedora 38 Workstation, client version 4.1.0.11250

When I attempt to place a file or folder within the ownCloud folder, I get the following errors:

Not allowed because you don't have permission to add files in that folder
or
Not allowed because you don't have permission to add folders in that folder 

I can only upload files and folders using the web-based GUI.
![image](https://github.com/owncloud/ocis/assets/61365551/0015aad4-7fe7-4db7-b93b-70576027252f)

**Comment by micbar** [comment]
@ThePatrickHe In order to help you, we would need more info about your setup.

Can you post your Server Version and the way how you deployed it? Logfiles from the server would also help.

**Comment by ThePatrickHe** [comment]
I am new to all of this-can you please help me figure out which logfiles you would need?

I simply followed the OCIS documentation for setting up a docker configuration. Here is my command:

```bash
docker run \
    --name ocis_runtime \
    --detach \
    --restart=always \
    -it \
    -p 9200:9200 \
    --mount type=bind,source=/home/patrick/5TB_HDD/ocis/ocis-config,target=/etc/ocis \
    --mount type=bind,source=/home/patrick/5TB_HDD/ocis/ocis-data,target=/var/lib/ocis \
    -e OCIS_INSECURE=true \
    -e PROXY_HTTP_ADDR=0.0.0.0:9200 \
    -e OCIS_URL=https://192.168.1.24:9200 \
    owncloud/ocis
```

where 5TB_HDD is a symlink to my external HDD in /mnt

I pulled the latest version of OCIS from docker hub. Is that what you are referring to by Server Version?

Thank you!

**Comment by ThePatrickHe** [comment]
Problem solved: I had setup the application using the V2.10.1 .rpm client available for Fedora in GNOME Software. After I realized this, I switched to the latest client available from ownCloud's website. While this fixed a previous issue I had with the sync stopping every ~5 minutes, this introduced the new issue described in this thread. After nuking ocis-data and ocis-config and deleting everything and starting over I was able to get everything working.
